### DISCUSSION TITLE
Negation of EndsWith

### DISCUSSION THREAD

**Discussion by asolux** [MikeSchulze/gdUnit4Mono#17] [Q&A] [answered]
Hello, how can a negation be formed with the test `EndsWith`. Or is there still a `NotEndsWith` method missing in `StringAssert`?

**Comment by MikeSchulze** [comment, marked as answer]
Hello, I am back from vacation therefore the late answer.

You can use `NotContains` instead. `NotEndsWith` is currently not provided.
Please note GdUnit4Mono is offical not released yet.

**Reply by asolux** [reply]
No problem, hope the vacation was nice.

Is not so bad I just thought I missed something obvious since everything else has a negation.

**Reply by MikeSchulze** [reply]
if you miss somthing feel free to create a feature request :)
